src/codeAnalysis/Rules.ruleset

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<RuleSet Name="PowerToys Rules" Description="Based on Microsoft Minimum Recomended Rules and customized to include other required rules. These rules focus on the most critical problems in your code, including potential security holes, application crashes, and other important logic and design errors. It is recommended to include this rule set in any custom rule set you create for your projects." ToolsVersion="15.0">
+<RuleSet Name="PowerToys Rules" Description="Based on Microsoft Minimum Recommended Rules and customized to include other required rules. These rules focus on the most critical problems in your code, including potential security holes, application crashes, and other important logic and design errors. It is recommended to include this rule set in any custom rule set you create for your projects." ToolsVersion="15.0">
```

Comment:
All files under the `codeAnalysis` folder should not be changed.

Reply:
Can you please help me get the tool that is responsible for this fixed?

Reply:
It's part of Visual Studio.